LogGenerator/src/main/java/com/loggenerator/LogGenerator.java

```diff
@@ -0,0 +1,166 @@
+        }
+        return sb.toString();
+    }
+
+    private static void grammarPrepare(Random random) throws InterruptedException {
+        for (int i = 0; i < 100; i++) {
+            String cpf = generateCPF(random);
+            String action = ACTIONS[random.nextInt(ACTIONS.length)];
+            String message = MESSAGESWCPF[i%5];
+            String to_user = generateCPF(random);
+            String id = generateRandomId();
+            String fileName = FILE_NAMES[new Random().nextInt(FILE_NAMES.length)];
+            String content = CONTENTS[new Random().nextInt(CONTENTS.length)];
+            String name = NAMES[new Random().nextInt(NAMES.length)];
+            String amount = "$" + String.valueOf((random.nextInt(99901) + 10) * 100) + ",00";
+            String date = generateDate(random);
+            String hours = generateHours(random);
+
+            message = message.replace("{CUSTOMER_ID}", id).replace("{TIME}", hours).replace("{DATE}", date).replace("{AMOUNT}", amount).replace("{USER_NAME}", name).replace("{SENDER_NAME}", name).replace("{CUSTOMER_NAME}", name).replace("{CPF}", "<s>" + cpf + "</s>").replace("{CONTENT}", content).replace("{FILE_NAME}", fileName).replace("{ID}", id).replace("{TO_USER}", to_user).replace("{ACTION}", action).replace("{DATETIME}", DATE_FORMAT.format(new Date()));
+
+            System.out.println(message);
+        }
+    }
+
+    public static void main(String[] args) throws InterruptedException {
+        System.setOut(new CustomPrintStream(System.out));
+
+        Random random = new Random();
+        grammarPrepare(random);
+        System.out.println("Start");
+        for (int i = 0; i < 100; i++) {
+
+            String cpf = generateCPF(random);
+            String action = ACTIONS[random.nextInt(ACTIONS.length)];
+            String message = MESSAGES[random.nextInt(MESSAGES.length)];
+            String to_user = generateCPF(random);
+            String id = generateRandomId();
+            String fileName = FILE_NAMES[new Random().nextInt(FILE_NAMES.length)];
+            String content = CONTENTS[new Random().nextInt(CONTENTS.length)];
+            String name = NAMES[new Random().nextInt(NAMES.length)];
+            String amount = "$" + String.valueOf((random.nextInt(99901) + 10) * 100) + ",00";
+            String date = generateDate(random);
+            String hours = generateHours(random);
+
+            message = message.replace("{CUSTOMER_ID}", id).replace("{TIME}", hours).replace("{DATE}", date).replace("{AMOUNT}", amount).replace("{USER_NAME}", name).replace("{SENDER_NAME}", name).replace("{CUSTOMER_NAME}", name).replace("{CPF}", cpf).replace("{CONTENT}", content).replace("{FILE_NAME}", fileName).replace("{ID}", id).replace("{TO_USER}", to_user).replace("{ACTION}", action).replace("{DATETIME}", DATE_FORMAT.format(new Date()));
+
+            System.out.println(message);
+            TimeUnit.SECONDS.sleep(1);
+        }
```

Comment:
Is this portion of code doing the same thing as the `grammarPrepare` method above?